cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityTopicOperator.java

```diff
@@ -277,7 +283,9 @@ public static EntityTopicOperator fromCrd(Reconciliation reconciliation, Kafka k
         varList.add(buildEnvVar(ENV_VAR_FULL_RECONCILIATION_INTERVAL_MS, Integer.toString(reconciliationIntervalMs)));
         varList.add(buildEnvVar(ENV_VAR_ZOOKEEPER_SESSION_TIMEOUT_MS, Integer.toString(zookeeperSessionTimeoutMs)));
         varList.add(buildEnvVar(ENV_VAR_TOPIC_METADATA_MAX_ATTEMPTS, String.valueOf(topicMetadataMaxAttempts)));
+        varList.add(buildEnvVar(ENV_VAR_SECURITY_PROTOCOL, securityProtocol));
```

Comment:
Given the value is always the same, maybe we can just use `DEFAULT_SECURITY_PROTOCOL` here and just remove the `securityProtocol` field?

Reply:
Should be fixed.